File: src/MyHealth.Client.Cordova/content/app/modules/shared/models/Doctor.ts
module MyHealth.Client.Cordova.Application.Shared {

    export class Doctor {
        address: string;
        currentRoomNumber: number;
        description: string;
        doctorId: number;
        email: string;
        id: string;
        mobile: string;
        name: string;
        patientCount: number;
        phone: string;
        picture: string;
        speciality: string;
        synchronized: boolean;
        tenantId: number;

        deserialize(data: any) {
            this.address = data.address;
            this.currentRoomNumber = data.currentRoomNumber;
            this.description = data.description;
            this.doctorId = data.doctorId;
            this.email = data.email;
            this.id = data.id;
            this.mobile = data.mobile;
            this.name = data.name;
            this.patientCount = data.patientCount;
            this.phone = data.phone;
            this.picture = data.picture;
            this.speciality = data.speciality;
            this.synchronized = data.synchronized;
            this.tenantId = data.tenantId;
            return this;
        }
    }
}

File: src/MyHealth.MobileApp/DataObjects/Doctor.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace MyHealth.MobileApp.DataObjects
{
    public class Doctor : CustomDataEntity
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.None)]
        public int DoctorId { get; set; }

        public string Name { get; set; }

        public string Address { get; set; }

        public string Description { get; set; }

        public string Phone { get; set; }

        public string Mobile { get; set; }

        public string Email { get; set; }

        public byte[] Picture { get; set; }

        public int TenantId { get; set; }

        public Specialities Speciality { get; set; }

        public int CurrentRoomNumber { get; set; }

        public int PatientCount { get; set; }

        public bool Synchronized { get; set; }

        public ICollection<Patient> Patients { get; set; }

    }
}